src/pip/_internal/operations/install/wheel.py

```diff
@@ -59,12 +59,11 @@
     from typing import Protocol
 
     class File(Protocol):
-        src_record_path = None  # type: RecordPath
-        dest_path = None  # type: str
-        changed = None  # type: bool
```

Comment:
The default values to None might need to be reverted, meaning that the type annotation was wrong. In any case probably safer to fix the type annotation here, and then if necessary update the default value in a separate PR.

Reply:
https://mypy.readthedocs.io/en/stable/class_basics.html#instance-and-class-attributes

Reply:
Yeah, this `attr = None  # type: MyType` is a special case because you can’t do `attr: MyType` before 3.6 (or something like that).

Reply:
@uranusjr Thanks for the clarification!